modules/nw-ingresscontroller-change-external.adoc

```diff
@@ -0,0 +1,41 @@
+// Module included in the following assemblies:
+//
+// * networking/ingress-operator.adoc
+
+[id="nw-ingresscontroller-change-external_{context}"]
+= Configuring the Ingress Controller endpoint publishing scope to External
+
+When a cluster administrator installs a new cluster without specifying that the cluster is private, the default Ingress Controller is created with a `scope` set to `External`.
+
+The Ingress Controller's scope can be configured to be `Internal` during installation or after, but cluster administrators can change an `Internal` Ingress Controller to `External`.
```

Comment:
I think the "but" should be an "and" or similar.

Reply:
Done!